infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutContextData.kt

```diff
@@ -53,8 +54,10 @@ sealed class LayoutContextData<T> : LayoutContextAware<T> {
     @get:JsonIgnore
     open val designRowId: DomainId<T>? get() = null
 
+    override val branch get() = designId?.let(LayoutBranch::design) ?: LayoutBranch.main
+
     @get:JsonIgnore
-    open val designId: DomainId<LayoutDesign>? get() = null
+    open val designId: IntId<LayoutDesign>? get() = null
```

Comment:
Tämän ei tarvitse koskaan olla arvuuttelutyyppiä

Reply:
Arvuuttelutyyppikommentti hieman hämäsi kun tämähän käytännössä on kuitenkin optional, mutta oletettavasti viittasit `DomainId`:hen etkä kysymysmerkkiin?

Reply:
Joo juuri näin